src/west/app/main.py

```diff
@@ -815,6 +852,24 @@ def do_run(self, args, ignored):
                          'which may be causing this issue.\n'
                          '  Try running "west update" or fixing the manifest.')
 
+class Alias(WestCommand):
+    # An alias command, it does not run itself
+
+    def __init__(self, cmd, args):
+        super().__init__(cmd, f'"{args}"', f'An alias that expands to "{args}"')
```

Comment:
Don't "naive" quote because the output maybe invalid, example:
```
aliases:
                     "list -f "{name} {revision}""
```
```suggestion
        super().__init__(cmd, f'"{args}"', f'An alias that expands to {args}')
```
Either quote correctly using `shlex`, or (much simpler and preferred) just don't quote here.